hydra-node/src/Hydra/Chain.hs

```diff
@@ -184,6 +186,7 @@ newtype Chain tx m = Chain
   -- reasonable local view of the chain and throw an exception when invalid.
   --
   -- Does at least throw 'PostTxError'.
+  , draftTx :: (IsChainState tx, MonadThrow m) => UTxOType tx -> m (Either (PostTxError tx) tx)
```

Comment:
Should: document this new method and in particular what types of exceptions are thrown.